bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/JwtHelper.java

```diff
@@ -78,13 +77,14 @@ private RsaJsonWebKey generateNewKey() throws JoseException, FileNotFoundExcepti
 
         String keyJson = newKey.toJson(OutputControlLevel.INCLUDE_PRIVATE);
 
-        IOUtils.write(keyJson, new FileOutputStream(file));
+        Files.writeString(file.toPath(), keyJson, StandardCharsets.UTF_8);
         return newKey;
     }
 
     private RsaJsonWebKey loadOrGenerateKey() throws FileNotFoundException, JoseException, IOException {
         try {
-            List<String> lines = IOUtils.readLines(new FileInputStream(KEY_FILE_PATH));
+            File file = new File(KEY_FILE_PATH);
+            List<String> lines = Files.readAllLines(file.toPath(), StandardCharsets.UTF_8);
             return (RsaJsonWebKey) JsonWebKey.Factory.newJwk(lines.get(0));
```

Comment:
We only need the first line. Could be changed to:

```suggestion
            final BufferedReader reader = Files.newBufferedReader(file.toPath());
            return (RsaJsonWebKey) JsonWebKey.Factory.newJwk(reader.readLine());
```